1001-2000/LIT1799Lifeof.xml

```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+        <abstract>
+            <p>A collection of homilies for each month of the year
+               describing the life and death of John the Baptist.
+                 It seems to be a rare composition, and it
+                  is different from a work of a similar structure
+                 contained in <ref type="mss" corresp="ESehpp007"/>, and another one
+                 in the edition
+                 <bibl><ptr target="bm:bm:Tarraqan2003GadlaYohannes"/></bibl>.</p>
+        </abstract>
          <langUsage><language ident="en">English</language></langUsage>
       </profileDesc>
+
       <revisionDesc>
          <change who="PL" when="2016-03-21"> Created file from google
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
-      <body>
-         <ab/>
+     <body>
+         <div type="edition">
+             <div type="textpart" subtype="incipit" xml:lang="gez">
+                 <note>The following incipit is taken from
+                       <ref type="mss" corresp="ESmy023"/>.</note>
+                 <ab>
+                   <pb n="4ra"></pb>
+                   በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቀዱስ፡ ፩አምላክ፡ ንዌጥን፡ በርድኤተ፡ እግዚአብሔር፡
+                   ጸሒፈ፡ ዜናሁ፡ ወመንክራቲሁ፡ ለቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ዘከመ፡ እፎ፡ ኮነ፡ ሕይወቱ፨
+                   ወከመ፡ ኮነ፡ ምትረተ፡ ርእሱ፡ በመዋዕለ፡ ሄሮድስ፡ ርጉም፡ ወዓማፂ፨ ወናሁ፡ ይቤ፡ ከመዝ፡ ዮሐንስ፡ አፈ፡
+                   በትረ፡ ያርክ፡ ሊቀ፡ ጳጳሳት፡ ዘሃገረ፡ ቅስጥንጥያ፨ ወሶበ፡ ኮነ፡ ስደቱ፡ ውስተ፡ አብራክይስ፡ አመ፡ ተስእሎ፡
+                   ፩አይሁዳዊ፡ ኬልቀደናዊ፡ ወውእቱ፡ ዘኮነ፡ ምእመነ፡ በእግዚእነ፡
+                 </ab>
+             </div>
+             <div type="textpart" subtype="explicit" xml:lang="gez">
+                 <note>The following explicit is taken
+                   from <ref type="mss" corresp="ESmy023"/>.</note>
+                 <ab>
+                   <pb n="77vb"></pb>
+                   ወሶበ፡ ኮኖ፡ ፵ወ፯፡ ዓመተ፡ አዕረፈት፡ አመ፡ ፪ለመስከረም፡ ወነፍሱሂ፡ እምድኅረ፡ ነበረት፡ ርእሱ፡
+                    ፲ወ፪፡ ዓመተ፡ አዕረፈት፡ አመ፡ ፲ወ፭፡ ለሚያዝያ፡
+                   ወከመዝ፡ ፈጸመ፡ ሠናየ፡ ሑረቱ፡ ብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ወአውገዙ፡ መምሕራነ፡
+                   ቤተ፡ ክርስቲያን፡ ከመ፡ <pb n="78ra"></pb>
+                    ምግባራተ፡ ሥጋ፡ ወኢምንተኒ፡ በዕለተ፡ በዓሉ፡ ለቅዱስ፡
+                    ዮሐንስ፡ መጥመቅ፨ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡ አግብሪሁ፡ አቡነ፡ ኪዳነ፡
+                    ወልድ፡ ወለተ፡ ግብርኤል፡ ለዓ፡ ዓለ፡ አ።
+                 </ab>
+                 </div>
+         </div>
+         <div type="bibliography">
+         </div>
+         <div type="bibliography">
+             <listRelation>
+                 <relation name="saws:isRelatedTo" active="LIT1799Lifeof" passive="PRS5741Johnthe"/>
+               <relation name="lawd:hasAttestation" active="LIT1799Lifeof" passive="EAPrqm029"/>
+               <relation name="lawd:hasAttestation" active="LIT1799Lifeof" passive="ESmy023"/>
+
```

Comment:
you need relations for the different texts here, with their IDs

Reply:
I used to indicate only shelfmarks, this was ok, was it? RQM-029 is not described yet.

Reply:
it is not at all necessary or requested to have hasAttestation relation for works, as these shall be linked as msItems anyway

what I meant you need relations for the texts which are similar but different to disambiguate